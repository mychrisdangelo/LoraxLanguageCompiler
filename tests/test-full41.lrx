/* 
 * Author: Chris D'Angelo
 * Test for mod operator
 */

int main()
{
	print("should print 2 = ", 7 mod 5, "\n");
}