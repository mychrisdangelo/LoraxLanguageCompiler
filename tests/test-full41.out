should print 2 = 2
